doc/index/Doxyfile

```diff
@@ -1,17 +1,131 @@
-# Doxyfile 1.8.1.2
+# Doxyfile 1.8.17
+
+# This file describes the settings to be used by the documentation system
+# doxygen (www.doxygen.org) for a project.
+#
+# All text after a double hash (##) is considered a comment and is placed in
+# front of the TAG it is preceding.
+#
+# All text after a single hash (#) is considered a comment and will be ignored.
+# The format is:
+# TAG = value [value, ...]
+# For lists, items can also be appended using:
+# TAG += value [value, ...]
+# Values that contain spaces should be placed between quotes (\" \").
+
```

Comment:
The modifications in this file are not needed. Or am I missing something?

Reply:
This is again automatic, this file is generated after upgrading to Doxyfile version 1.8.17.
The additions are almost all comments (probably manually removed from previous versions ?) but there are some extra options added and some old ones removed. 
For example the [`PERL_PATH`](https://github.com/boostorg/geometry/blob/develop/doc/index/Doxyfile#L295) which seems to be deprecated; see https://github.com/doxygen/doxygen/commit/6d1535c38fe6bdaa2a00fff0e7e43774a740a4ce